src/eg/Cargo.toml

```diff
@@ -7,15 +7,18 @@ publish = false
 [dependencies]
 digest = "0.10"
 hmac = "0.12"
+itertools = "0.12"
 lazy_static = "1.4"
 num-bigint = "0.4"
 num-traits = "0.2"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 sha2 = "0.10"
 static_assertions = "1.1.0"
+thiserror = "1.0"
 util = { path = "../util" }
 base64 = "0.21.2"
+chrono = { version = "0.4.34", features = ["serde"] }
```

Comment:
Earlier versions of EGDS 2.0 specified a date, but after discussions we softened that language due to the crazy complexity datetimes bring in a specification document. As much as I'm a fan of `chrono`, I'd really prefer that the core `eg` crate not depend on any date/time library.
We'll take it in this PR, but let's make a plan to move it to the 'info' string soon.